src/xbot/common/math/PIDManager.java

```diff
@@ -19,6 +19,7 @@
     private DoubleProperty maxOutput;
     private DoubleProperty minOutput;
     private BooleanProperty isEnabled;
+    private boolean isIMasked = false;
```

Comment:
I feel like `mask` is slightly ambiguous, I don't get an immediate sense of "masked on" or "masked off". 

What about `isISupressed` or some other verb more closely associated with hiding/suppressing/disabling?